runtime/doc/options.txt

```diff
@@ -6762,8 +6776,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	displayed when 'verbosefile' is set.
 
 						*'viewdir'* *'vdir'*
-'viewdir' 'vdir'	string	(default for MS-DOS and Win32: "$VIM/vimfiles/view",
-				 for Unix: "~/.vim/view")
+'viewdir' 'vdir'	string	(default: "$XDG_DATA_HOME/nvim/view")
```

Comment:
I know we have discussed this before in #78, but I’m still wondering if we are correct in defining view session files, backups, swap files etc. as **essential** data going into `$XDG_DATA_HOME` instead of as non-essential data, which would go into `$XDG_CACHE_HOME` [according to the spec](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html).

I have had some doubts since my last [contribution to that discussion](https://github.com/neovim/neovim/issues/78#issuecomment-39668459) (which applied to the inadvisability of using `$XDG_RUNTIME_DIR`, a directory that _must_ not survive log offs or reboots), but now that this feature is nearing release, I would be remiss in not bringing up my suspicion that we might be off base in our interpretation of the XDG spec. I know I should have brought this up earlier (especially as it has [been brought up in #78](https://github.com/neovim/neovim/issues/78#issuecomment-77776048), a fact I seem to have completely missed), but are we absolutely certain that  `$XDG_DATA_HOME` isn’t me at for things like user installed spelling dictionaries and such, while state should better go into `$XDG_CACHE_HOME`?